components/RegisterForm/index.tsx

```diff
@@ -48,9 +36,22 @@ const RegisterForm: React.FC<RegisterFormProps> = (props) => {
         secureTextEntry
         containerStyle={styles.inputs}
       />
+      <Input
+        placeholder="Verify Password"
+        leftIcon={
+          <MaterialIcons name="lock" size={20} color="grey" />
+        }
+        onChangeText={setVerify}
+        value={verify}
+        secureTextEntry
+        containerStyle={styles.inputs}
+      />
       <Button
         title="Create Account"
-        onPress={onRegister}
+        onPress={() => {
+          if (onRegister && verify === password)
+            onRegister(email, password);
```

Comment:
Missing curly braces for the same reason as screens/LoadingScreen/index.tsx

Reply:
sure